app/src/main/kotlin/com/wire/android/ui/common/imagepreview/AvatarPickerFlow.kt

```diff
@@ -40,7 +40,7 @@ fun rememberPickPictureState(): AvatarPickerFlow {
     }
     val onChosenPictureUri = getShareableAvatarUri(context)
     val takePictureFLow = rememberTakePictureFlow(
-        shouldPersistUri = { wasSaved ->
+        onPictureTaken = { wasSaved ->
             if (wasSaved) {
                 scope.launch { postProcessCapturedAvatar(onChosenPictureUri, context) }
```

Comment:
I think like we discussed today we could move such thingies into the ViewModel, including settings the pictureState I think, it would also allow to survive such post processing during the configuration changes